client/signup/controller.js

```diff
@@ -245,6 +245,20 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
 
+		// See: 1113-gh-Automattic/experimentation-platform for details.
```

Comment:
Code looks great. But I'm wondering if we can find a better place for this part and do something like in https://github.com/Automattic/wp-calypso/blob/7e6cbb6e1042fd06feb796c60afff16dd60103a0/client/signup/config/flows-pure.js#L198 ?